src/main/java/io/lighty/yang/validator/formats/Analyzer.java

```diff
@@ -42,12 +42,12 @@ private void printOut() {
         }
     }
 
-    private void analyzeSubstatement(final DeclaredStatement subStatement) {
+    private void analyzeSubstatement(final DeclaredStatement<?> subStatement) {
         String name = subStatement.statementDefinition().getStatementName().getLocalName();
         counter.compute(name, (key, val) -> (val == null) ? 1 : val + 1);
-        final Collection substatements = subStatement.declaredSubstatements();
+        final Collection<?> substatements = subStatement.declaredSubstatements();
```

Comment:
this is not correct: declaredSubstatements() returns a properly-bound collection -- no need to do a ton of these typecasts!

Reply:
Done